<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1348</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
border:4px;
background-color:#FFEEE7;
border-color:#666666;
border-radius:4px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1351</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget:pane {
border:4px;
background-color:#FFEEE7;
border-color:#666666;
border-radius:4px;
}</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des colis</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Statistiques">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>320</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_export_2">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>220</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Exporter Excel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>341</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Ajouter Colis</string>
      </property>
      <widget class="QLabel" name="label_numcolis">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Numéro de la colis</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dateliv">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de livraison</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_adresseliv">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Adresse de livraison</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_expediteur">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom de l'expéditeur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_destinataire">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom du destinataire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numcolis">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adresseliv">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_expediteur">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_destinataire">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouterColis">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>320</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_livraison">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_recherche_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>40</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>520</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>520</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>1210</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_colis">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>571</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des partenaires</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>391</width>
        <height>451</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Nouveau Partenaire</string>
      </property>
      <widget class="QLabel" name="label_nbC">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Nombre de commandes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouterP">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>330</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbC">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_ID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_produit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Type de produit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_produit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_nom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Nom du partenaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_recherche">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supp">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>510</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>510</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_retour">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>10</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_export">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>300</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Exporter Excel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stat">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_part">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>120</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1348</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
